packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/readme.md

```diff
@@ -197,23 +198,3 @@ export * from './flight/index';
 
 You can now use departureDateTimeConsideringTimezone to access the timezone information.
 See [utils.Date](https://github.com/AmadeusITGroup/otter/blob/main/packages/%40ama-sdk/core/src/fwk/date.ts) for more information.
-
-### How to keep the timezone (prevent Date replacement with utils.Date)
```

Comment:
I think there's a section in `packages/@ama-sdk/showcase-sdk/readme.md` that needs to be updated as well.